backend/Application/Jobs/JobsBackgroundService.cs

```diff
@@ -69,14 +73,18 @@ private async Task RunJob(TJob job, CancellationToken token)
             }
             _logger.Information($"{job.GetUniqueIdentifier()} starts.");
             
+            using var _ = TraceContext.StartActivity(job.GetUniqueIdentifier());
+            using var __ = LogContext.PushProperty("Job", job.GetUniqueIdentifier());
```

Comment:
Either I need to do like this or I need to add indention like 

```
            using(TraceContext.StartActivity(job.GetUniqueIdentifier()))
            {
                using (LogContext.PushProperty("Job", job.GetUniqueIdentifier()))
                {
                    // ....
                }
                
            };
``` 

I would very much like to avoid the indention since it really makes it more difficult to read the code.